### DISCUSSION TITLE
Integrate security into CI pipeline

### DISCUSSION THREAD

**Discussion by emregency** [directus/directus#3739] [Feature Requests]
Wouldn't that be cool if we had some security checks built-in to the pipelines? 

[Snyk ](https://support.snyk.io/hc/en-us/articles/360004032117-GitHub-integration) to check package vulnerabilities

[DeepCode ](https://deepcode.freshdesk.com/support/solutions/articles/60000346380-how-do-i-get-started-with-github-) for ML-based static code analysis

I am using them regularly and can only recommend.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
